codux.config.json

```diff
@@ -1,52 +1,65 @@
 {
-  "$schema": "https://wixplosives.github.io/codux-config-schema/codux.config.schema.json",
-  "boardGlobalSetup": "./_codux/board-global-setup.ts",
-  "componentsDiscovery": {
-    "include": ["src/**"],
-    "exclude": ["_codux/component-templates/**"]
-  },
-  "boardsPath": "_codux/boards",
-  "newComponent": {
-    "componentsPath": "src/components",
-    "templatesPath": "_codux/component-templates"
-  },
-  "safeRender": {
-    "maxInstancesPerComponent": 1000
-  },
-  "scripts": {
-    "install": {
-      "title": "Install",
-      "description": "Run install",
-      "command": "npm i",
-      "trigger": ["checkout", "pull", "setup"]
+    "$schema": "https://wixplosives.github.io/codux-config-schema/codux.config.schema.json",
+    "boardGlobalSetup": "./_codux/board-global-setup.ts",
+    "componentsDiscovery": {
+        "include": [
+            "src/**"
+        ],
+        "exclude": [
+            "_codux/component-templates/**"
+        ]
+    },
+    "boardsPath": "_codux/boards",
+    "newComponent": {
+        "componentsPath": "src/components",
+        "templatesPath": "_codux/component-templates"
+    },
+    "safeRender": {
+        "maxInstancesPerComponent": 1000
+    },
+    "scripts": {
+        "install": {
+            "title": "Install",
+            "description": "Run install",
+            "command": "npm i",
+            "trigger": [
+                "checkout",
+                "pull",
+                "setup"
+            ]
+        }
+    },
+    "styling": {
+        "solution": "css"
+    },
+    "styleFilesConfig": {
+        "commonStyleFilePattern": "**/src/styles/**"
+    },
+    "resolve": {
+        "alias": {
+            "@styles": "./src/styles",
+            "@styles/*": "./src/styles/*",
+            "/*": "./*",
+            "~/*": "./src/*",
+            "node:fs": false,
+            "node:fs/promises": false,
+            "node:path": false,
+            "node:crypto": false,
+            "crypto": false,
+            "node:stream": false,
+            "stream": false,
+            "node:os": false,
+            "node:util": false,
+            "node:events": false,
+            "node:url": false,
+            "node:buffer": false,
+            "node:assert": false
+        }
+    },
+    "svgLoader": "both",
+    "externalProviders": {
+        "wix": {
+            "projectId": "2e7b526c-2b9c-40bb-bb77-05dc1a0dc29a"
```

Comment:
I assume that you connected to your personal meta site for dev purposes. If yes, please revert the changes to the codux.config.json